wp-job-manager-functions.php

```diff
@@ -638,12 +640,8 @@ function job_manager_dropdown_categories( $args = '' ) {
 		$r['pad_counts'] = true;
 	}
 
-	// WPML & Polylang caching per language
-	if ( defined( 'ICL_SITEPRESS_VERSION' ) ) {
-		$r['lang'] = apply_filters( 'wpml_current_language', NULL );
-	} elseif ( function_exists( 'pll_current_language' ) ) {
-		$r['lang'] = pll_current_language();
-	}
+	/** This filter is documented in wp-job-manager.php */
+	$r['lang'] = apply_filters( 'wpjm_lang', null );
 
 	extract( $r );
```

Comment:
I know this is standard WordPress to use `extract`, but this makes me want to hulk smash all over this code (not that you added it) :)